deskew/cli.py

```diff
@@ -24,7 +24,7 @@ def main() -> None:
     options = parser.parse_args()
 
     image = io.imread(options.input)
-    grayscale = rgb2gray(image)
+    grayscale = image if len(image) == 2 else rgb2gray(image)
```

Comment:
it should be ` grayscale = image if len(image.shape)<3 else rgb2gray(image)`

Reply:
I agree that the `.shape` is missing, but the image shape can't have a length of 1 then the `== 2` working, not?

Reply:
Sure